src/utils/address.ts

```diff
@@ -76,4 +79,21 @@ export async function deriveAddressP2SH(index: number, storage: IStorage): Promi
     throw new Error('No multisig data');
   }
   return deriveAddressFromDataP2SH(multisigData, index, storage.config.getNetwork().name);
-}
\ No newline at end of file
+}
+
+/**
+ * Parse the public key and return an address.
+ *
+ * @param pubkey Hex string conveying the public key.
+ * @returns The address object from parsed publicKey or null if publicKey is nulish.
+ */
+export function getAddressFromPubkey(pubkey: string): Address|null {
+  if (pubkey == null) {
+    return pubkey;
+  }
+
+  const network = Config.getNetwork();
+  const pubkeyBuffer = hexToBuffer(pubkey);
+  const base58 = new bitcoreAddress(bitcorePublicKey(pubkeyBuffer), network.bitcoreNetwork).toString()
+  return new Address(base58, { network });
+}
```

Comment:
issue(blocking): move network to arguments.

This will make the util not dependent on the current configured config, but the caller will decide which network to use.

Reply:
Done at de23637844ad7e2f03e114efd79b2b94f51d989a